PluralKit.Bot/Handlers/ReactionAdded.cs

```diff
@@ -108,7 +110,7 @@ private async ValueTask TryHandleProxyMessageReactions(MessageReactionAddEvent e
                 }
 
             case "\U0001F514": // Bell
-            case "\U0001F6CE": // Bellhop bell
+            case "\U0001F6CE\U0000fe0f": // Bellhop bell
```

Comment:
I'd rather that the variation selector gets stripped off entirely before we check these, instead of adding it to the checks whenever Discord break something!

I'd adjust the `evt.Emoji.Name` in the `switch` statement at the top of these checks to something like `evt.Emoji.Name.Split("\U0000fe0f", 2)[0]` so it always just gets the character(s) before the variation selector, if there is one.

Reply:
Hmm, do we need to worry about discord adding in other variation selectors? IE, should I just grab the first unicode id no matter what the second is? Or might that cause more issues down the line?